From 13bef6441fe483ae2f28a8668bd2d25291631a26 Mon Sep 17 00:00:00 2001
From: Claudia Pellegrino <claui@users.noreply.github.com>
Date: Sun, 23 Oct 2022 18:03:54 +0200
Subject: [PATCH] Remove `test` modules from sdist, wheel

I think the `test` modules are not supposed to be included in the sdist
and wheels.
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 7d0856a..77cc03e 100644
--- a/setup.py
+++ b/setup.py
@@ -21,7 +21,7 @@
     long_description_content_type='text/markdown',
     url='https://github.com/evil-mad/ink_extensions',
 
-    packages=find_packages(exclude=['contrib', 'docs', 'tests']),
+    packages=find_packages(exclude=['contrib', 'docs', 'test', 'test.*']),
     install_requires=[
         'lxml'
     ],
